src/resources/authoring/tutorial.schema.json

```diff
@@ -54,7 +54,7 @@
                             },
                             "location": {
                               "description": "A specific location on screen for the popup",
-                              "type": "string"
+                              "pattern": "(top-left|top-center|top-right|middle-left|middle-center|middle-right|bottom-left|bottom-center|bottom-right)"
```

Comment:
I like the use of the pattern! Seems like you could have left the "type" in as well, but I guess it's implied.